test/language/expressions/generators/params-trailing-comma-length.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2016 Jeff Morrison. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: Check that trailing commas are permitted in function argument lists
```

Comment:
Similar for other descriptions, which should be a bit more descriptive about the individual test's high level purpose